examples/toast.html

```diff
@@ -160,10 +160,20 @@ <h1>Open Window Handler</h1>
                 // Note: This manual check for query string is for backwards compatibility of this
                 // deployed example.  In your own apps, you can assume the query param will be
                 // provided by Genesys Cloud if you have configured it in your app's config.
-                let envQueryParamName = 'pcEnvironment';
-                if (window && window.location && typeof window.location.search === 'string' &&
-                    window.location.search.indexOf(envQueryParamName) >= 0) {
-                    myClientApp = new window.purecloud.apps.ClientApp({pcEnvironmentQueryParam: envQueryParamName});
+                const envQueryParamName = 'pcEnvironment';
+                const hostQueryParamName = 'gcHostOrigin';
+                const targetEnvQueryParamName = 'gcTargetEnv';
+                const locationSearch = (window && window.location && typeof window.location.search === 'string') ? window.location.search : '';
+                const queryParams = new URLSearchParams(locationSearch);
+                if (queryParams.get(hostQueryParamName) && queryParams.get(targetEnvQueryParamName)) {
```

Comment:
we can probably assume/enforce that both of these will be provided in this example code as well.

could switch this to an `||`